modules/avcs/api/getAvcsRepositoryInfo.ts

```diff
@@ -22,6 +22,7 @@ export function getAvcsRepositoryInformation(network: SupportedNetworks): Reposi
     page: 'delegates/templates'
   };
 
-  const avcsRepositoryInfo = network === SupportedNetworks.MAINNET ? repoMainnet : repoTest;
+  const avcsRepositoryInfo =
+    network === SupportedNetworks.MAINNET || network === SupportedNetworks.TENDERLY ? repoMainnet : repoTest;
```

Comment:
If the plan is to only support Mainnet and Tenderly I think this can just point to the mainnet repo (no need for test repo). Same for the delegates repo in the following file